src/index.html

```diff
@@ -1,13 +1,65 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div data-qa="card" class="container">
+      <img
+        class="container__photo"
+        src="images/imac.jpeg"
+        alt="imac"
+       >
+
+      <h2 class="container__title">
+        Моноблок APPLE A1419 iMac<br>27" Retina 5K (MNED2UA/A)
+      </h2>
+
+      <p class="container__sku">
+        Код товара: 195434
+      </p>
+
+      <div class="container__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="container__reviews">
+          Отзывов: 5
+        </p>
+      </div>
+
+      <div class="container__pricefield">
+        <p class="container__pricetext">
+          Цена:
+        </p>
+
+        <p class="container__price">
+          69 999 грн
+        </p>
+      </div>
+
+
+      <button class="container__button" name="purchase">
```

Comment:
```suggestion
      </div>

      <button class="container__button" name="purchase">
```
Не потрібні лишні пусті лінії